src/lasp_transmission_instrumentation.erl

```diff
@@ -40,9 +40,9 @@
 -include("lasp.hrl").
 
 %% State record.
--record(state, {type,
+-record(state, {entity,
                 tref,
-                size=0,
+                size_per_type=orddict:new(),
```

Comment:
The idea is to count the number of bytes sent per type:

- `aae_send`
- `delta_ack`
- `delta_send`
- `broadcast`
- ...